### DISCUSSION TITLE
Y Axis formatting to show Millions / Billions

### DISCUSSION THREAD

**Discussion by RajaPoseidon** [perspective-dev/perspective#2606] [Q&A]
hi,

i am looking for a way in y line graph to show numbers in Billions ( like 180 B ) . right now i did applied custom column to divide the number by 1000000000 and name the viewer as number in billions , but would like to know if their a way to show the "B" lable on the axis itself?

**Comment by aszenz** [comment]
I'm facing a similar issue while the numbers can be formatted now in the datagrid plugin (using number format compact option) but this setting has no effect on chart plugins.
